kratos/solving_strategies/builder_and_solvers/builder_and_solver.h

```diff
@@ -222,6 +222,8 @@ class BuilderAndSolver
 
         mReshapeMatrixFlag = false; //by default the matrix is shaped just once
         //		mVectorsAreInitialized = false;
+
+        mEchoLevel = 0;
```

Comment:
just as a suggestion, but it would be best to directly put the =0 where the mEchoLevel is initialized

Reply:
I think it is better to initialize it.

Reply:
Ok